app/src/app.py

```diff
@@ -26,5 +26,4 @@ def load_user(user_id):
 def index():
     return render_template('index.html')
 
-db.create_all()
-
+db.create_all()
```

Comment:
Throws an error regarding application context for latest version of Flask_SQLAlchemy 3.0; would suggest nesting within a `with app.app_context()` block (as mentioned here: https://stackoverflow.com/questions/73961938/flask-sqlalchemy-db-create-all-raises-runtimeerror-working-outside-of-applicat)

Reply:
Sorry, never mind—downgrading to the specified flask and flask_login versions specified in requirements.txt fixes these issues. Would still recommend adding the above to cover compatibility for future flask versions, in the case we ever end up updating any libraries down the line in the future, but not necessary.